micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/custom/ResponseRethrowingErrorHandler.groovy

```diff
@@ -29,7 +52,15 @@ class ResponseRethrowingErrorHandler implements ResponseErrorHandler {
 
     protected String getLoggedErrorResponseBody(ClientHttpResponse response) {
         String responseBody = getResponseBody(response)
-        log.error("Response error: status code [$response.statusCode], headers [$response.headers], body [$responseBody]")
+
+        switch (errorMessageLogging) {
+            case ErrorMessageLogging.ERROR:
+                log.error("Response error: status code [$response.statusCode], headers [$response.headers], body [$responseBody]")
+                break
+            case ErrorMessageLogging.WARN:
+                log.warn("Response error: status code [$response.statusCode], headers [$response.headers], body [$responseBody]")
+                break
```

Comment:
Maybe add explicitly a `default` path with a comment that there is nothing to do?